ui/admin/app/components/form/worker/create-worker-led/index.js

```diff
@@ -17,19 +17,34 @@ class Tag {
 export default class FormWorkerCreateWorkerLedComponent extends Component {
   // =services
   @service features;
+  @service browserObject;
 
   // =attributes
   @tracked generatedWorkerAuthToken;
-  @tracked clusterId;
+  @tracked clusterId = this.clusterIdFromURL;
```

Comment:
This wouldn't change, so it doesn't need to be tracked.  In the template, you can access `this.clusterIdFromURL` directly.
```suggestion

```

Reply:
But again, don't I need this tracked value to still keep the input working? We still want the input especially in cases where this fails to extract a value. This is to just to set an initial value for the input.